Brought back 'index.js only' necessity (#252)
kottans/frontend: tasks/js-post-oop.md

When complete do the following:
1. You will require code review for this task:
   - If you are a **p2p course** student, please,
   follow [these instructions](https://github.com/kottans/frontend-2019-p2p/blob/master/CONTRIBUTING.md)
   - If you are a student of the **offline** Kottans course, please,
   follow [these instructions](https://github.com/kottans/frontend-2019-homeworks/blob/master/README.md)

>    - Note, that especially this task you require to put a single file (**index.js only**) in your homework repo.

Reviewer: ```suggestion
   - Note, that especially this task requires you to submit a single file (`index.js` **only**) to the `frontend-2019-homeworks` or `frontend-2019-p2p` repo for code review.
```
